cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -601,9 +602,248 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
+              |The following query shows an example of how to grant this privilege:""".stripMargin)
+          query("GRANT EXECUTE PROCEDURE db.schema.* ON DBMS TO procedureExecutor", ResultAssertions((r) => {
+            assertStats(r, systemUpdates = 1)
+          })) {
+            p("Users with the role 'procedureExecutor' can then run any procedure in the `db.schema` namespace. The procedure will be run using the users own privileges.")
+            statsOnlyResultTable()
+          }
+          p("The resulting role should have privileges that only allow executing procedures in the `db.schema` namespace:")
+          query("SHOW ROLE procedureExecutor PRIVILEGES", assertPrivilegeShown(Seq(Map()))) {
+            p("Lists all privileges for role 'procedureExecutor'")
+            resultTable()
+          }
+
+          p(
+            """If we want to allow executing all but a few procedures, we can grant `EXECUTE PROCEDURES *` and deny the unwanted procedures.
+              |For example, the following queries allows for executing all procedures except `dbms.killTransaction` and `dbms.killTransactions`:""".stripMargin)
+          query("GRANT EXECUTE PROCEDURE * ON DBMS TO deniedProcedureExecutor", ResultAssertions((r) => {
+            assertStats(r, systemUpdates = 1)
+          })) {
+            statsOnlyResultTable()
+          }
+          query("DENY EXECUTE PROCEDURE dbms.killTransaction* ON DBMS TO deniedProcedureExecutor", ResultAssertions(r => {
+            assertStats(r, systemUpdates = 1)
+          })) {
+            statsOnlyResultTable()
+          }
+          p("The resulting role should have privileges that only allow executing all procedures except `dbms.killTransaction` and `dbms.killTransactions`:")
+          query("SHOW ROLE deniedProcedureExecutor PRIVILEGES", assertPrivilegeShown(Seq(Map()))) {
+            p("Lists all privileges for role 'deniedProcedureExecutor'")
+            resultTable()
+          }
+        }
+
+        section("The `EXECUTE BOOSTED PROCEDURE` privilege", "boosted-execute-procedure-subsection", "enterprise-edition") {
+          p(
+            """The ability to execute a procedure with elevated privileges can be granted via the `EXECUTE BOOSTED PROCEDURE` privilege.
+              |A user with this privilege is allowed to execute the procedures matched by the <<procedure-name-globbing, name-globbing>>
+              |without the execution being restricted to their other privileges.
+              |The following query shows an example of how to grant this privilege:""".stripMargin)
+          query("GRANT EXECUTE BOOSTED PROCEDURE db.labels, db.relationshipTypes ON DBMS TO boostedProcedureExecutor", ResultAssertions((r) => {
+            assertStats(r, systemUpdates = 2)
+          })) {
+            p(
+              """Users with the role 'boostedProcedureExecutor' can then run `db.labels` and `db.relationshipTypes` with full privileges,
```

Comment:
Same as above with the sentence bisecting the query and result
![Screenshot 2020-09-23 at 20 19 46](https://user-images.githubusercontent.com/12032446/94059358-33ae9a00-fdda-11ea-8574-a40363eeadd4.png)